docs/content/getting-started/Configuring for IDE.md

```diff
@@ -0,0 +1,26 @@
+---
+category: getting-started
+categoryindex: 1
+index: 2
+---
+
+# Configuring for the IDE
+
+## Visual Studio Code
+
+In order to configure analyzers for VSCode, you will need to update your project's `.vscode/settings.json` file or your user settings. You should only need the following settings:
```

Comment:
@TheAngryByrd, where can people find the log if there's an SDK mismatch? Users may use older analyzers, which might not be detected by the tooling. I believe we mention this somewhere. I suggest including an explanation on this page about where this information will be displayed.

Reply:
Briefly looking at the code, I'm not sure we actually log this correctly.

Added issue here: https://github.com/ionide/FsAutoComplete/issues/1350

Reply:
Alright, thanks, @1eyewonder maybe we just link to that issue in the docs.